### ISSUE TITLE
fix: Dismissal tooltip in dialog header does not show, nor can we override the name

### ISSUE THREAD

**Issue by adamint** [microsoft/fluentui-blazor#3237]
# 🐛 Bug Report

`FluentDialogHeader` does not have settable tooltip text that will be shown, so it is impossible to localize this value. Additionally, I haven't been able to get the tooltip to show up at all (including in the Customized Dialog sample). 

![Image](https://github.com/user-attachments/assets/cb6774d0-38ec-4638-b145-2a7db2e43142)

## 💻 Repro or Code Sample

Try hovering over a dialog header in the `Customized Dialog` sample - the tooltip does not show even though the default parameter value for `ShowDismissTooltip` is true.

## 💁 Possible Solution

A parameter `DismissTooltipText` could be added to control the text content, in addition to fixing the visibility bug.

## 🔦 Context

this is an accessibility gap in the component

**Comment by dvoituron** [comment]
You can use the `DialogParameters.DismissTitle` to define the tooltip.

```csharp
private async Task OpenDialogAsync()
{
    DialogParameters parameters = new()
        {
            Title = $"Hello {simplePerson.Firstname}",
            PrimaryAction = "Yes",
            PrimaryActionEnabled = false,
            SecondaryAction = "No",
            DismissTitle = "Hello",                  // 👈
            Width = "500px",
            TrapFocus = _trapFocus,
            Modal = _modal,
            PreventScroll = true
        };

    IDialogReference dialog = await DialogService.ShowDialogAsync<SimpleDialog>(simplePerson, parameters);
    DialogResult? result = await dialog.Result;

```

![Image](https://github.com/user-attachments/assets/ee156581-cb35-4646-89d5-0801b078d611)

**Comment by vnbaaij** [comment]
@dvoituron we added a new thing because of the title being focused, right? Could that be why the tooltip isn't showing?

**Comment by dvoituron** [comment]
I don't think so. We added a flag to not display the tooltip automatically when the dialog is opened. But when using the mouse the tooltip must appear.

**Comment by dvoituron** [comment]
Can you confirm me that it works with this parameter or not? 😀

**Comment by adamint** [comment]
It does not

**Comment by dvoituron** [comment]
Do you have an example of code to reproduce this issue?

In my previous comment, I added `DismissTitle = "Hello"` in the Demo Sample and the tooltip was displayed.

**Comment by vnbaaij** [comment]
@dvoituron it can be reproduced with https://www.fluentui-blazor.net/Dialog#customizeddialog

I get the tooltip to show if I add `UseTooltipService="false"` in https://github.com/microsoft/fluentui-blazor/blob/dbc81d5c1f4559285284726d0008c44a6ce3bbc3/src/Core/Components/Dialog/FluentDialogHeader.razor?plain=1#L30

**Comment by dvoituron** [comment]
Weird. 

When using the Customized Dialog, the tooltip HTML code is created in the **TooltipProvider**, but it doesn't respond to the mouse event. Could this be due to a mismatch in the HTML rendering order of the code?

I've just created the PR #3241 to correct this problem, disabling the Tooltip Service only for this dialog close button.
In the meantime, if this is acceptable for your project, you can disable the **TooltipProvider** as follows:

```csharp
// Program.cs
builder.Services.AddFluentUIComponents(options =>
{
    options.UseTooltipServiceProvider = false;
});
```

And remove the `<FluentTooltipProvider />` from you Layout page.
